sys/include/crypto/ciphers.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
 #elif defined(CRYPTO_AES)
     #define CIPHER_MAX_CONTEXT_SIZE CIPHERS_MAX_KEY_SIZE
 #else
-    // 0 is not a possibility because 0-sized arrays are not allowed in ISO C
+// 0 is not a possibility because 0-sized arrays are not allowed in ISO C
```

Comment:
Minor nitpick and I know it is not your fault, but do you mind changing this comment from C++ style to C style?

Reply:
Done.